docs/documentation/models/react.md

```diff
@@ -30,36 +30,100 @@ Hook for accessing LLM chat data and state.
 declare function useChat(options?: UseChatOptionsType): UseChatResultType;
 ```
 
-### Usage
+For a more detailed example, see the guide [Building client applications](/guides/models/building-client-applications.md).
+
+### Options
 
 ```ts
-const { input, setInput, messages, setMessages, loading, error, onChange, onSubmit } = useChat();
+/**
+ * The options supplied to the useChat hook.
+ */
+type UseChatOptionsType = {
+  /**
+   * The API endpoint to call when submitting a new message.
+   *
+   * Defaults to `/api/chat`.
```

Comment:
Adjacent to this PR, but I was wondering if having no default, and throwing if not passed is a more intuitive and solid design.

Reply:
This particular parameter default was influenced by another package, but I think I would agree it's better to not assume a default here. Leaving for now since it's already a part of the lib, but can remove it in a major release (since it would be a breaking change).